Makefile

```diff
@@ -999,30 +999,30 @@ ldb: tools/ldb.o $(LIBOBJECTS)
 INSTALL_PATH ?= /usr/local
 
 uninstall:
-	rm -rf $(INSTALL_PATH)/include/rocksdb \
-	  $(INSTALL_PATH)/lib/$(LIBRARY) \
-	  $(INSTALL_PATH)/lib/$(SHARED4) \
-	  $(INSTALL_PATH)/lib/$(SHARED3) \
-	  $(INSTALL_PATH)/lib/$(SHARED2) \
-	  $(INSTALL_PATH)/lib/$(SHARED1)
+	rm -rf $(DESTDIR)$(INSTALL_PATH)/include/rocksdb \
```

Comment:
A bit of personal taste and GNU Coding Standards: https://www.gnu.org/prep/standards/html_node/DESTDIR.html. It also eases interfacing with the Debian build. Not doing so forces builders to customize their builds (which are usually adjusted after the above standards).